cinema/views.py

```diff
@@ -112,27 +136,52 @@
 
         return MovieSessionSerializer
 
+    def perform_create(self, serializer):
+        if not self.request.user.is_staff:
+            raise PermissionDenied(
+                "You do not have permission to perform this action."
+            )
+        serializer.save()
+
+    def perform_update(self, serializer):
+        if not self.request.user.is_staff:
+            raise PermissionDenied(
+                "You do not have permission to perform this action."
+            )
```

Comment:
Raising a `PermissionDenied` exception directly in the `perform_update` method might lead to inconsistent behavior. Consider using permission classes to handle these checks more uniformly.